data-reporting-service/src/main/resources/db/sp-person-001.sql

```diff
@@ -1,4 +1,4 @@
-CREATE or ALTER PROCEDURE dbo.sp_Person_Event  @user_id_list varchar(max)
```

Comment:
We are not maintaining or using sp_Person_Event at this point.

Reply:
It is just for reference purpose. Will remove it in a future PR